index.bs

```diff
@@ -122,17 +125,18 @@ Use-Cases and Requirements {#usecases-requirements}
 Use cases {#usecases}
 ---------
 
-    1. A 3D compass web application monitor's the device's orientation and aligns the compass heading in a three-dimensional space.
-    1. A web-based game uses the device's orientation relative to the Earth's surface as user input. For example, simulation of Ball-in-a-maze puzzle.
-    1. A WebVR API polyfill implementation tracks the headset's (or mobile device's for mobile VR) orientation.
-    1. A mapping web application orientates the 2D map with the orientation of the device.
+    1. A compass monitor's the device's orientation relative to true north and aligns the compass heading in a three-dimensional space.
```

Comment:
Generally, I think made-up use cases that aren't rooted in _real problems_ faced by _real developers_ in _real life_ provide a false sense of security. I recommend avoiding them.